File: web/src/types/seller.ts
export interface Seller {
  id: string;
  firstName: string | null;
  lastName: string | null;
  displayName: string;
  grade: string | null;
  teacher: string | null;
  notes: string | null;
  picklistBarcode: string;
  createdAt: string;
  updatedAt: string;
}

export interface CreateSellerRequest {
  firstName?: string | null;
  lastName?: string | null;
  displayName: string;
  grade?: string | null;
  teacher?: string | null;
  notes?: string | null;
}

File: api/src/HamptonHawksPlantSales.Core/Models/Seller.cs
namespace HamptonHawksPlantSales.Core.Models;

public class Seller : BaseEntity
{
    public string? FirstName { get; set; }
    public string? LastName { get; set; }
    public string DisplayName { get; set; } = string.Empty;
    public string PicklistBarcode { get; set; } = string.Empty;
    public string? Grade { get; set; }
    public string? Teacher { get; set; }
    public string? Notes { get; set; }
}
